basics/indexing/star-tree-index.md

```diff
@@ -1,16 +1,22 @@
+---
+description: Learn what a star-tree index is and the concept of how one works.
+---
+
 # Star-Tree Index
 
+In this page you will learn what a star-tree index is and gain a conceptual understanding of how one works.
+
 Unlike other index techniques which work on a single column, the star-tree index is built on multiple columns and utilizes pre-aggregated results to significantly reduce the number of values to be processed, resulting in improved query performance.
 
 One of the biggest challenges in realtime OLAP systems is achieving and maintaining tight SLAs on latency and throughput on large data sets. Existing techniques such as [sorted index](forward-index.md) or [inverted index](inverted-index.md) help improve query latencies, but speed-ups are still limited by the number of documents that need to be processed to compute results. On the other hand, pre-aggregating the results ensures a constant upper bound on query latencies, but can lead to storage space explosion.
 
-Here we introduce the **star-tree** index to utilize the pre-aggregated documents in a smart way to achieve low query latencies but also use the storage space efficiently for aggregation/group-by queries.
+Here we introduce the **star-tree** index to utilize the pre-aggregated documents in a way that achieves low query latencies but also uses the storage space efficiently for aggregation and group-by queries.
 
 {% embed url="https://www.youtube.com/watch?v=bwO0HSXguFA" %}
 
-### Existing solutions
+## Existing solutions
```

Comment:
sounds like we're using the terms "solutions," "approaches," and "technique" interchangeably, also, the type of index is referred to as an approach....what terminology is best to use in this context?

Reply:
good question. as I was reading it the first time, what I thought I read was that solutions were more general, like a fix to a problem that might be taken care of using any of multiple approach options. with that, a technique would be the way an approach was applied...but I may have been reading into it.